File: app/src/main/java/com/babestudios/companyinfouk/ui/officerdetails/OfficerDetailsActivityView.java
package com.babestudios.companyinfouk.ui.officerdetails;

import net.grandcentrix.thirtyinch.TiView;
import net.grandcentrix.thirtyinch.callonmainthread.CallOnMainThread;

interface OfficerDetailsActivityView extends TiView {

	@CallOnMainThread
	void showProgress();

	@CallOnMainThread
	void hideProgress();
}

File: app/src/main/java/com/babestudios/companyinfouk/ui/officerdetails/OfficerDetailsActivityView.kt
package com.babestudios.companyinfouk.ui.officerdetails

import net.grandcentrix.thirtyinch.TiView
import net.grandcentrix.thirtyinch.callonmainthread.CallOnMainThread

interface OfficerDetailsActivityView : TiView {

	@CallOnMainThread
	fun showProgress()

	@CallOnMainThread
	fun hideProgress()
}